### PR TITLE
game: Track missing vars for 'ClientHitboxMaxZ' in antilag

### PR DESCRIPTION
**Pull request by Vorschreibung**
These were missing in antilag markers, but are now required for calculating hitboxes correctly.

### CODE REVIEW

**Comment by Vorschreibung** [comment]
@ryzyk-krzysiek Please review :pray:

File: src/game/g_antilag.c [hunk 1]
```diff
@@ -314,6 +336,10 @@ static qboolean G_AdjustSingleClientPosition(gentity_t *ent, int time)
 			ent->torsoFrame.pitchAngle    = ent->client->clientMarkers[j].torsoPitchAngle;
 			ent->torsoFrame.yawing        = ent->client->clientMarkers[j].torsoYawing;
 			ent->torsoFrame.pitching      = ent->client->clientMarkers[j].torsoPitching;
+			if (ent->client->clientMarkers[j].torsoAnimationMovetype)
```
Comment (ryzyk-krzysiek) [inline src/game/g_antilag.c:339]: I guess these checks could be incorrect if `movetype 0` is a valid move type (I don't know if it is). Maybe checking if `ent->torsoFrame.animation` is valid pointer is better, or assuming that these pointers are always valid (`ClientHitboxMaxZ` does not check if it is valid so if it is valid there it should be valid here too I think).

Comment (Vorschreibung) [reply src/game/g_antilag.c:339]: 0 is `ANIM_MT_UNUSED` - should never be the case
I'll add the pointer check tho, thanks!

Comment (ryzyk-krzysiek) [reply src/game/g_antilag.c:339]: Then I have no further remarks, everything looks correct.

**Pull request merged**